niryo_one_commander/scripts/robot_commander.py

```diff
@@ -105,19 +119,24 @@ def execute_command(self, cmd):
             status = CommandStatus.SUCCESS # todo get status from send_tool_command
             message = "Tool command has been successfully processed"
         else: # move command
-            if cmd_type == CommandType.JOINTS:
-                self.arm_commander.set_joint_target(cmd.joints)
-            elif cmd_type == CommandType.POSE:
-                self.arm_commander.set_pose_target(cmd.position.x, cmd.position.y, cmd.position.z,
-                                                   cmd.rpy.roll, cmd.rpy.pitch, cmd.rpy.yaw)
-            elif cmd_type == CommandType.POSITION:
-                self.arm_commander.set_position_target(cmd.position.x, cmd.position.y, cmd.position.z)
-            elif cmd_type == CommandType.RPY:
-                self.arm_commander.set_rpy_target(cmd.rpy.roll, cmd.rpy.pitch, cmd.rpy.yaw)
-            elif cmd_type == CommandType.SHIFT_POSE:
-                self.arm_commander.set_shift_pose_target(cmd.shift.axis_number, cmd.shift.value)
-        
-            status, message = self.compute_and_execute_plan()
+            if cmd_type == CommandType.EXECUTE_TRAJ:
```

Comment:
As with the comment on arm_moveit_commander.py, 
the "plan" used in the code and the "plan" here are not the same thing. Maybe use cmd.plan.trajectory here, or rename the message ?
Just to avoid confusion in the future